contrib/examples/elixir/lib/brod_sample/application.ex

```diff
@@ -7,8 +7,11 @@ defmodule BrodSample.Application do
 
   def start(_type, _args) do
     children = [
-      BrodSample.GroupSubscriber
+      BrodSample.GroupSubscriberV2
+      # FIXME: If start both process only the "BrodSample.GroupSubscriberV2" consumes messages.
```

Comment:
Any ideia of how to solve it?

Reply:
It solved my problem: https://github.com/kafka4beam/brod/pull/522/commits/3f92dc69d7e0129fc2737cbfbdc02523bc5f50f6